README.md

```diff
@@ -1,214 +1,201 @@
+communicate with external systems. These adaptors are used by
+[OpenFn Lightning](https://github.com/OpenFn/lightning) and the
+[OpenFn CLI](https://github.com/openfn/cli) for workflow automation.
+
+# Table of Contents
+
+- [Quick Start](#quick-start)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [Using Adaptors](#using-adaptors)
+  - [With OpenFn CLI](#with-openfn-cli)
+  - [With OpenFn Lightning](#with-openfn-lightning)
+- [Contributing](#contributing)
+- [Development](#development)
+  - [Creating a New Adaptor](#creating-a-new-adaptor)
+    - [Prerequisites](#prerequisites-1)
+    - [Development Steps](#development-steps)
+    - [Testing Documentation Changes](#testing-documentation-changes)
+    - [Best Practices](#best-practices)
+- [Changesets](#changesets)
+- [Versioning](#versioning)
+- [Releases](#releases)
+- [Pre-releases](#pre-releases)
+- [Metadata](#metadata)
+- [Useful Resources](#useful-resources)
+
+## Quick Start
+
+### Prerequisites
+
+- [asdf](https://github.com/asdf-vm/asdf) installed globally
+- `Node.js` and `pnpm` managed via asdf
+
+### Setup
+
+```bash
+# Install asdf plugins
 asdf plugin add nodejs https://github.com/asdf-vm/asdf-nodejs.git
 asdf plugin-add pnpm
-```
 
-Then:
-
-```
-asdf install # Install tool versions
+# Install dependencies and build
+asdf install
 pnpm install
 pnpm build
 pnpm run setup
 ```
 
-## Running scripts
+## Using Adaptors
 
-Every repo provides a common set of npm scripts:
+### With OpenFn CLI
 
-To run them for all scripts in `packages`, call
-`pnpm --filter "./packages/** <script>`.
+1. Install the CLI globally:
 
-For example:
+   ```bash
+   npm install -g @openfn/cli
+   ```
 
-```
-pnpm --filter "./packages/**" build
-pnpm --filter "./packages/**" test
-```
+2. Create a job file (e.g., `job.js`):
 
-## Contributing
+   ```javascript
+   // Example: Send data to Salesforce
+   create('Account', {
+     Name: $.data.company_name,
+     Industry: $.data.industry,
+   });
+   ```
 
-### Assign yourself to an issue
+3. Run the job:
+   ```bash
+   openfn job.js -a salesforce -s tmp/state.json -o tmp/output.json
+   ```
 
-Read through the existing [issues](https://github.com/OpenFn/adaptors/issues)
-and assign yourself to the issue you have chosen. If anything needs
-clarification, don't hesitate to leave a comment on the issue and we will get
-back to you as soon as possible.
+### With OpenFn Lightning
 
-If there isn't already an issue for the feature you would like to contribute,
-please make one and assign yourself.
+Lightning is our open-source workflow automation platform that uses these
+adaptors. See the
+[Lightning documentation](https://github.com/OpenFn/lightning/blob/main/RUNNINGLOCAL.md#using-local-adaptors)
+for setup and usage instructions.
 
-### Open a pull request
+## Contributing
 
-1. Clone the adaptors repository, then
-   [fork it](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+1. **Find an issue**: Browse
```

Comment:
Maybe. A lot of users will be coming here because they want to add their own adaptor, or will have something specific they want to fix.